book/src/main/resources/static/src/pages/BookList.tsx

```
@@ -0,0 +1,86 @@
+          </tr>
+        </thead>
+        <tbody>
+          {books.map((book, index) => (
+            <tr key={book.isbn || index}>
+              <td>
+                {book.image && <img src={book.image} alt={book.title} style={{width: '50px', height: '70px', objectFit: 'cover'}} />}
+              </td>
+              <td>
+                <Link to={`/book/${book.isbn}${window.location.search}`} dangerouslySetInnerHTML={{__html: book.title}} />
+              </td>
+              <td dangerouslySetInnerHTML={{__html: book.author}} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)